gcc/rust/rust-session-manager.cc

```diff
@@ -53,7 +53,32 @@ const char *kHIRDumpFile = "gccrs.hir.dump";
 const char *kHIRTypeResolutionDumpFile = "gccrs.type-resolution.dump";
 const char *kTargetOptionsDumpFile = "gccrs.target-options.dump";
 
-const std::string kDefaultCrateName = "example";
+const std::string kDefaultCrateName = "rust_out";
+
+static std::string
+infer_crate_name (const std::string filename)
+{
+  if (filename == "-")
+    return kDefaultCrateName;
+
+  std::string crate = std::string (filename);
+#ifdef __WIN32
+  size_t path_sep = crate.find_last_of ('\\');
+#else
+  size_t path_sep = crate.find_last_of ('/');
+#endif
```

Comment:
> There's a `file_separator` constant somewhere in the parser :D I think it would help here. It's located [here in rust-system.h](https://github.com/Rust-GCC/gccrs/blob/master/gcc/rust/rust-system.h#L64-L70)

Ah, thank you for the pointer! I will use that instead.